WG.md

```diff
@@ -8,12 +8,43 @@ for review by the TSC at an agreed upon date. A WG will be automatically disband
 
 ## Current Working Groups
 
+### Generics and circuit families
+
+**Objective**: ...
+
+Chair: Niel de Beaudrap  
+Members: Colm Ryan, Andrew Cross, Ali Javadi, Blake Johnson
+
+### Grammar
+
+**Objective**: 
+
```

Comment:
Some of the **Questions** to ask in this group that comes to my mind:

* Are their pain points in the grammar?
* Are their places we can improve?
* Can we improve the performance?
* Can we improve the style?